addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -388,10 +386,6 @@
             <field name="mode">primary</field>
             <field name="model">mailing.mailing</field>
             <field name="arch" type="xml">
-                <xpath expr="//form" position="attributes">
-                    <attribute name="js_class">mailing_mailing_view_form_full_width</attribute>
```

Comment:
Why exactly do we remove this? Are there many issues with keeping it?

If it's just because the snippets menu doesn't work well, we can easily fix that. Especially since that js class is only used here.